### ISSUE TITLE
Add "buy me a coffe" link in the README file

### ISSUE THREAD

**Issue by fribse** [xaviml/controllerx#75]
This is catastrophic, there is no 'Buy me a coffee link' ANYWHERE!!!

**Comment by tobimori** [comment]
lol

**Comment by xaviml** [comment]
You really got me there @fribse. :rofl: I was already thinking: "What did I break already?" hahaha

I have been meaning to add the link a long time ago, but I've never got around to do it. I will do it asap now! :stuck_out_tongue_closed_eyes: 

Thanks! :)

P.S.: I will change the title if you do not mind :laughing:

**Comment by tobimori** [comment]
You can use the option to display a sponsor button next to the watch button by creating a FUNDING.yml in .github

**Comment by xaviml** [comment]
Thanks @tobimori, I will do :smiley:

**Comment by xaviml** [comment]
Hi guys :) I just added the Sponsor link into the projects through FUNDING.yml file as @tobimori suggested and I also added a badge in the README. The sponsor link is this [one](https://www.buymeacoffee.com/xaviml).

I just want to state here that I started doing this project for myself since it was something I faced in my own setup. Since the HA community has created plenty of side projects to help each other out, I wanted to do the same to contribute back by sharing this project. So what pays off my work with ControllerX is having a wide variety of side projects and the Home Assistant project itself.

Thank you :)
